### ISSUE TITLE
[request] Make an "Advanced compression" menu to choose from, the same as CloneZilla's

### ISSUE THREAD

**Issue by rado84-github** [rescuezilla/rescuezilla#353]
Make a menu "Advanced compression" in RZ from which we can choose any of these compression methods with the same descriptions that you can see in CloneZilla. The "p" methods are the most important bc they're designed to use all the cores and threads of the CPU which immensely reduces the time needed to create the backup. Nowadays almost everyone has a multicore configuration and so nobody will want to wait 20 minutes for something that can be done in under 3 minutes. That's exactly the time the z9p method does things - 40GB is being compressed into 2.5 times smaller backup in less than 3 minutes.

![Clonezilla-compressions](https://user-images.githubusercontent.com/41172201/183854239-a2ff9834-b1db-4c70-bf62-3a94f74b113f.png)

The current compression settings of RZ are way too unclear what they are. Every time I need to use RZ for whatever reason, I end up waiting at least 20 minutes, sometimes even more, which is why I still prefer to use CZ, whenever I can, to RZ.

**Comment by rado84-github** [comment]
If you could add them the way they're written in CZ (z1, z2, z9, z9p), that would be awesome! My favorite compression is z9p. :)

**Comment by rado84-github** [comment]
Any approximation as to when these options will be available in RZ?

**Comment by llvs** [comment]
The parallel options are still unavailable in rescuezilla-2.4.2-64bit.jammy as of today. Just tried it.

**Comment by rado84-github** [comment]
Yeah, I don't imagine this is a top priority for the developer, so I doubt we'll see them anytime soon.
